### PR TITLE
Prometheus VPA

### PR DESCRIPTION
**Pull request by NathanFlurry**

### CODE REVIEW

**Comment by NathanFlurry** [comment]
@MasterPtato can you give this a close look & test it on staging to make sure it's solid?

**Comment by MasterPtato** [comment]
@NathanFlurry you want me to deploy this to staging2?

**Comment by MasterPtato** [comment]
@NathanFlurry lmk on slack when you get the credentials

**Comment by MasterPtato** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #222** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/222?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #398** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/398?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rivet-gg/rivet/222?utm_source=stack-comment).

**Comment by MasterPtato** [comment]
@NathanFlurry mb i turned this into a PR by adding to the stack

**Comment by MasterPtato** [comment]
@NathanFlurry confirmed VPA working, had to update the prometheus count to 2 since the VPA can't scale otherwise

**Review by MasterPtato — APPROVED** [review]

File: infra/tf/k8s_infra/vpa.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+resource "helm_release" "vpa" {
+    count = var.limit_resources ? 1 : 0
+
+	name = "vpa"
+    repository = "https://charts.fairwinds.com/stable"
+	namespace = kubernetes_namespace.vpa[0].metadata.0.name
+	chart = "vpa"
+	version = "3.0.2"
+}
```
Comment (AngelOnFira) [inline infra/tf/k8s_infra/vpa.tf:17]: Nit: mixed tabs and spaces

```suggestion
resource "kubernetes_namespace" "vpa" {
	count = var.limit_resources ? 1 : 0

	metadata {
		name = "vpa"
	}
}

resource "helm_release" "vpa" {
	count = var.limit_resources ? 1 : 0

	name = "vpa"
	repository = "https://charts.fairwinds.com/stable"
	namespace = kubernetes_namespace.vpa[0].metadata.0.name
	chart = "vpa"
	version = "3.0.2"
}
```

**Review by AngelOnFira — APPROVED** [review]

**Pull request merged**